dissect/volume/md/md.py

```diff
@@ -13,71 +13,72 @@
 from dissect.volume.raid.raid import RAID, Configuration, PhysicalDisk, VirtualDisk
 from dissect.volume.raid.stream import Level
 
-DeviceDescriptor = Union[BinaryIO, "Device"]
+MDPhysicalDiskDescriptor = Union[BinaryIO, "MDPhysicalDisk"]
 
 
 class MD(RAID):
-    """Read an MD RAID set of one or multiple devices/file-like objects.
+    """Read an MD RAID set of one or multiple disks/file-like objects.
```

Comment:
The different between "device" and "disk" was intentional in the initial code, by the way. I used the nomenclature that was used in the respective technologies source code and documentation to align with that as much as possible. If we're going to change that around, maybe make a note of it in the docstrings somewhere (as it's most prevalent in the MD code). Something that MD usually refers to everything as a (physicai/virtual) device rather than a disk.

Reply:
Ah ok. As it was not really consistent I picked `disk`, as that was what vinum mostly uses, `device` being more the entry in `/dev`. I could do a `s/disk/device/` if that makes more sense.

Reply:
The way you have it now is good.